scripts/tests/unit/test_schema_cleaner.py

```diff
@@ -223,6 +223,10 @@ def test_field_defaults(self):
         cleaner.field_defaults({'field_details': field_details})
         self.assertEqual(field_details['doc_values'], False)
 
+        field_details = {**field_min_details, **{'type': 'wildcard'}}
```

Comment:
`index` is not in field_min_details, you should add it in the `{'type': 'wildcard'}` dictionary, if you want to test that it properly gets removed

Reply:
```suggestion
        field_details = {**field_min_details, **{'type': 'wildcard', 'index': True}}
```